detekt-core/src/main/kotlin/io/gitlab/arturbosch/detekt/core/suppressors/AnnotationSuppressor.kt

```
@@ -19,6 +19,9 @@ internal fun annotationSuppressorFactory(rule: Rule, bindingContext: BindingCont
         it.qualifiedNameGlobToRegex()
     }
     return if (annotations.isNotEmpty()) {
+        if (rule.isForbiddenSuppress()) {
+            return null
+        }
```

```suggestion
```

No need. If someone say on the detekt configuration fail that they want to suppress all the errors of `ForbiddenSuppress` on the functions annotated with `@Composable` that's their problem. I agree that adding an `@Suppress("ForbiddenSuppress")` can be "accidental" or could not be cached when doing a review. But if you change the configuration of a rule we should honor that.